packages/mds-agency/request-handlers.ts

```diff
@@ -414,65 +439,61 @@ export const submitVehicleTelemetry = async (req: AgencyApiRequest, res: AgencyA
       logger.info(name, 'no valid telemetry in', data.length, 'items:', body, 'failures:', fails)
       res.status(400).send({
         error: 'invalid_data',
-        error_description: 'none of the provided data was valid',
-        result: 'no valid telemetry submitted',
-        failures
+        error_description: 'None of the provided data was valid',
+        error_details: failures
       })
     }
   } catch (err) {
-    res.status(400).send({
-      error: 'invalid_data',
-      error_description: 'none of the provided data was valid',
-      result: 'no valid telemetry submitted',
-      failures: [`device_id ${data[0].device_id}: not found`]
+    res.status(500).send({
+      error: 'server_error',
```

Comment:
Should just send your agencyServerError boi here

Reply:
Nah cause it has the `error_details` that are special

Reply:
The `error_details` that are special are actually inaccurate for this code path, the original error message we sent was super off, and has actually lead to some diagnostic issues in the past